alacritty/src/event.rs

```diff
@@ -400,12 +393,17 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
     }
 
     #[cfg(not(windows))]
-    fn create_new_window(&mut self) {
+    fn create_new_window(&mut self, #[cfg(target_os = "macos")] tabbing_id: Option<String>) {
```

Comment:
Ah, the reason I did this is because they share the internal logic, otherwise we'll duplicate how it's done, windows is completely separate due to lack of IPC.